Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1110,6 +1092,10 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
         if geo is None:
           #If using geofilter on wireframed does not get wrppaed not sure why so sticking to many mappers
           act = vcs2vtk.doWrap(act,[x1,x2,y1,y2],wrap)
+        if mapper is missingMapper:
+            actors.append([act,missingMapper,[x1,x2,y1,y2]])
+        else:
+            actors.append([act,[x1,x2,y1,y2]])
```

Comment:
One path adds [actor, mapper, bounds], the other omits the mapper. Is this intentional?